terraform/envs/test/main.tf

```diff
@@ -23,6 +23,9 @@ module "upload-service-database" {
   db_username = "${var.db_username}"
   db_password = "${var.db_password}"
   db_instance_count = "${var.db_instance_count}"
+  subnet_id = "${element(data.aws_subnet_ids.vpc.ids, 0)}"
```

Comment:
I've been looking at this for a while now and I'm having a bit of trouble deciphering the naming here.  Is this vpc_id/subned_id for the entire upload service, the database or the load balancer?  If the former, it is correctly named.  Otherwise please can we indicate its purpose by calling it `db_subnet_id` or `db_lb_subnet_id` or something.  Same for `subnet_ids` and `vpc_id`.